docs/reference/setup/install/docker.asciidoc

```diff
@@ -0,0 +1,290 @@
+[[docker]]
+=== Install Elasticsearch with Docker
+
+Elasticsearch is also available as a Docker image.
+The image is built with https://www.elastic.co/guide/en/x-pack/current/index.html[X-Pack].
+
+==== Security note
+
+NOTE: https://www.elastic.co/guide/en/x-pack/current/index.html[X-Pack] is preinstalled in this image.
+Please take a few minutes to familiarize yourself with https://www.elastic.co/guide/en/x-pack/current/security-getting-started.html[X-Pack Security] and how to change default passwords. The default password for the `elastic` user is `changeme`.
+
+NOTE: X-Pack includes a trial license for 30 days. After that, you can obtain one of the https://www.elastic.co/subscriptions[available subscriptions] or https://www.elastic.co/guide/en/x-pack/current/security-settings.html[disable Security]. The Basic license is free and includes the https://www.elastic.co/products/x-pack/monitoring[Monitoring] extension.
+
+Obtaining Elasticsearch for Docker is as simple as issuing a +docker pull+ command against the Elastic Docker registry.
+
+ifeval::["{release-state}"=="unreleased"]
+
+However, version {version} of Elasticsearch has not yet been released, so no Docker image is currently available for this version.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+The Docker image can be retrieved with the following command:
+
+["source","sh",subs="attributes"]
+--------------------------------------------
+docker pull {docker-image}
+--------------------------------------------
+
+endif::[]
+
+[[docker-cli-run]]
+==== Running Elasticsearch from the command line
+
+[[docker-cli-run-dev-mode]]
+===== Development mode
+
+Elasticsearch can be quickly started for development or testing use with the following command:
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of the Elasticsearch Docker image has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+--------------------------------------------
+docker run -p 9200:9200 -e "http.host=0.0.0.0" -e "transport.host=127.0.0.1" {docker-image}
```

Comment:
No it won't work only with `http.host=0.0.0.0` (just tested it again), I presume because the shipped elasticsearch.yml in the image sets `network.host=0.0.0.0`